examples/basics/scene/stereo.py

```diff
@@ -8,6 +9,8 @@
 Example demonstrating stereo vision in a scene has an anisotropic aspect
 ratio. This example can be used to test that the cameras behave
 correctly with nested translated/rotated cameras.
+
+NOTE: This example is currently broken!
```

Comment:
I'd prefer to have it in another PR, having divided branches like this for
as long as we have is not good... but if you think it is critical we can
wait.